### PR TITLE
Commit blocker option - No warnings in transformer to publish

### PR DESCRIPTION
**Pull request by everton-rosario**
This PR:

* [x] Creates a flag in settings "block_publish_with_warnings"
* [x] Flag defaults to false (disabled)
* [x] If flag is true and transformer results in warnings, Instant Article is not registered 

Depends on SDK: https://github.com/facebook/facebook-instant-articles-sdk-php/pull/154

### CODE REVIEW

File: rules-configuration.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "selector" : "span"
     }, {
         "class": "ParagraphRule",
-        "selector": "p"
+        "selector": "pad"
```
Comment (diegoquinteiro) [inline rules-configuration.json]: that doesn't look right ;)

File: class-instant-articles-publisher.php [hunk 2]
```diff
@@ -89,6 +89,17 @@ public static function submit_article( $post_id, $post ) {
 					$dev_mode
 				);
 
+				// Don't process if contains warnings and blocker flag for transformation warnings is turned on.
+				if ( count( $adapter->transformer->getWarnings() ) > 0
+				  && isset( $publishing_settings['block_publish_with_warnings'] )
+					&& $publishing_settings['block_publish_with_warnings'] ) {
+
+					// Unpublishes if already published
+					$client->removeArticle( $article->get_canonical_url() );
```
Comment (diegoquinteiro) [inline class-instant-articles-publisher.php]: love this; let's prevent outdated posts to be live =)

**Comment by diegoquinteiro** [comment]
overall it is good, but please fix you test leftover =)

File: rules-configuration.json (hunk 1 above)
Comment (everton-rosario) [inline rules-configuration.json]: #fuuuu 
that was a test case. 
Tks for spotting it :-)

**Pull request merged**